src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AnnotationTypeMemberWriterImpl.java

```diff
@@ -269,7 +269,7 @@ private TypeMirror getType(Element member) {
     }
 
     public void addDefaultValueInfo(Element member, Content annotationContent) {
-        if (utils.isAnnotationType(member)) {
+        if (utils.isAnnotationInterface(member.getEnclosingElement())) {
```

Comment:
This looks like more than just a rename... but it also looks like a plausible bug fix.

Ah, this seems to go with the corresponding change in `Utils.isAnnotationInterface` which used to have a visitor that implicitly did the `getEnclosingElement`.

Reply:
Correct.